ui/articles-aem-renderer/src/main/java/org/godtools/articles/aem/db/ManifestAssociationRepository.java

```diff
@@ -0,0 +1,64 @@
+package org.godtools.articles.aem.db;
+
+import android.app.Application;
+import android.arch.lifecycle.LiveData;
+import android.os.AsyncTask;
+
+import org.godtools.articles.aem.model.Article;
+import org.godtools.articles.aem.model.ManifestAssociation;
+
+import java.util.List;
+
+/**
+ * This class is used a connector for the Data Access Object for Manifest Associations
+ */
+public class ManifestAssociationRepository {
+
+    private ManifestAssociationDao mManifestAssociationDao;
+
+    public ManifestAssociationRepository(Application application) {
```

Comment:
you can actually pass in any `Context` object, it doesn't need to be the Application object